code/app/build.gradle.kts

```diff
@@ -69,18 +69,13 @@ dependencies {
     implementation("com.google.android.material:material:1.3.0")
     implementation(project(":edgeidentity"))
 
-    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
-    implementation("com.adobe.marketing.mobile:identity:3.0.0-SNAPSHOT") {
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-    }
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion")
+    implementation("com.adobe.marketing.mobile:identity:3.0.0")
     implementation("com.adobe.marketing.mobile:edgeconsent:3.0.0-SNAPSHOT") {
         exclude(group = "com.adobe.marketing.mobile", module = "edge")
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-
     }
-    implementation("com.adobe.marketing.mobile:assurance:2.+")
+    implementation("com.adobe.marketing.mobile:assurance:3.0.0")
```

Comment:
if we change this to 3.0.0-SNAPSHOT we can merge into staging directly, just saying 👯